src/app/(auth)/_actions/uploadImage.ts

```diff
@@ -0,0 +1,25 @@
+﻿'use server'
+
```

Comment:
**nit:** uploading is not a part of `auth` domain. Let's do `app/(files)/_actions/uploadImage.ts`